static/js/navbar/__tests__/ui/Navbar.test.jsx

```diff
@@ -1,118 +1,269 @@
+                exist = true;
+            }
+        });
+        expect(exist).toBe(true);
+    });
+
+    test("Should render with action links", () => {
+        const wrapper = shallow(
+            <Navbar
+                logoUrl=""
+                tenantTitle=""
+                userName=""
+                links={[
+                    {
+                        id: "login",
+                        label: "Login",
+                        url: "/login"
+                    },
+                    {
+                        id: "logout",
+                        label: "Logout",
+                        url: "/logout"
+                    }
+                ]}
+                dropdowns={[]}
+                profileItems={[]}
+                isLoggedIn={false}
+                lightTheme
+            />
+        );
+        expect(wrapper.find({href: "/logout"}).length).toBe(1);
+        expect(wrapper.find({href: "/login"}).length).toBe(1);
+    });
+
+    test("Should render with profile items", () => {
+        const wrapper = mount(
+            <Navbar
+                logoUrl=""
+                tenantTitle=""
+                userName="Toto"
+                links={[]}
+                dropdowns={[]}
+                profileItems={[
+                    {
+                        id: "logout",
+                        label: "Logout",
+                        url: "/logout"
+                    }
+                ]}
+                isLoggedIn={false}
+                lightTheme
+            />
+        );
+        const dropdownLabel = wrapper.find(".tuiv2_dropdown__label");
+        expect(dropdownLabel.length).toBe(1);
+        expect(dropdownLabel.text()).toBe("Toto");
+    });
+
+    test("Should render with dropdowns", () => {
+        const wrapper = mount(
+            <Navbar
+                logoUrl=""
+                tenantTitle=""
+                userName="Toto"
+                links={[]}
+                dropdowns={[
+                    {
+                        type: "dropdown",
+                        id: "dropdownEx",
+                        label: "Dropdown",
+                        spanClass: "lnr-rocket",
+                        items: [
+                            {
+                                id: "item1",
+                                label: "Item 1",
+                                url: "#"
+                            },
+                            {
+                                id: "item2",
+                                label: "Item 2",
+                                url: "#"
+                            }
+                        ]
+                    }
+                ]}
+                profileItems={[]}
+                isLoggedIn={false}
+                lightTheme
+            />
+        );
+        const dropdownLabel = wrapper.find(".tuiv2_dropdown__label");
+        expect(dropdownLabel.length).toBe(1);
+        expect(dropdownLabel.text()).toBe("Dropdown");
```

Comment:
Les erreurs changent, on comprend mieux se qui se passe.
Je sais pas ce que pond jest mais ça pourrait être : 
expected "pouet" to be "dropdown" but it wasn't
vs
expect element <span class="tuiv2_dropdown__label"> to have text "dropdown" but it didn't

Pour le texte, c'est pas le plus flagrant mais pour d'autres matchers, c'est vachement mieux.
expected component to exist vs expected false to be true